docs/migration/guide/move-ref-docs.md

```diff
@@ -0,0 +1,130 @@
+   Page two
+   Page three
+
+> Release notes
+   > Known issues
+          APM Android Agent
+   > Breaking changes
+          APM Android Agent
+   > Deprecations
+          APM Android Agent
+   APM Android Agent release notes
+```
+
+This structure is defined in the repo's `docset.yml` file.
+
+For the **full Elastic.co/docs site**, the assembler references the individual content set definitions (`toc.yml`) within the repo and organizes the content accordingly:
+
+![Diagram of how everything maps together](../../images/great-drawing-of-new-structure.png)
+
+## How to Move Reference Content
+
+The steps below explain how to move reference content. You can also take a look at our [sample PR](https://github.com/elastic/apm-agent-android/pull/398), which has specific commits to illustrate some of the steps below.
+
+### Step 1: Delete Existing AsciiDoc Files
+
+:::{important}
+Skip this step for **any Cloud repos** and the **search-ui repository**.
+:::
+
+Ensure you only delete **external documentation files**.  
+- In some repositories, this means deleting the entire `/docs` directory.  
+- In others, manually verify which files should be removed.  
+- Use [`conf.yaml`](https://github.com/elastic/docs/blob/master/conf.yaml) to determine what should and shouldn't be removed.
+
+Example commit: [#398/commit](https://github.com/elastic/apm-agent-android/pull/398/commits/749803ae9bccdb9f8abdf27a5c7434350716b6c0)
+
+### Step 2: Copy and Paste New Content
+
+Move content from `asciidocalypse` to the correct directory in the target repo.  
+Use [issue#130](https://github.com/elastic/docs-eng-team/issues/130) to determine the correct path structure.
+
+Example commit: [#398/commit](https://github.com/elastic/apm-agent-android/pull/398/commits/3f966b0e1fa2f008da23d02f2c9e91a60c1bdf8d)
+
+### Step 3: Add the new CI checks
+
+**docs-build**
+* The file to add: https://github.com/elastic/docs-content/blob/main/.github/workflows/docs-build.yml
+* The path and name of the new file: `.github/workflows/docs-build.yml`
+
+**docs-cleanup**
+* The file to add: https://github.com/elastic/docs-content/blob/main/.github/workflows/docs-cleanup.yml
+* The path and name of the new file: `.github/workflows/docs-cleanup.yml`
+
+Example commit: [#398/commit](https://github.com/elastic/apm-agent-android/pull/398/commits/e869386bbf4af23d51432226f1fd3935d233e43d)
+
+### Step 4: Delete the asciidoc warning
+
+:::{important}
+Skip this step for **any Cloud repos** and the **search-ui repository**.
+:::
+
+During the migration freeze, we added a check to each repository that warned when a PR was opened against asciidoc files in `main`. It is now safe to remove this file.
+
+File to delete: `.github/workflows/comment-on-asciidoc-changes.yml`
+
+Example commit: [#398/commit](https://github.com/elastic/apm-agent-android/pull/398/commits/be422934e79c5ecadd7b76523d2e1676fc86f323)
+
+### Step 4: Wait for CI to Pass
+
+Verify that all automated checks pass before proceeding.
```

Comment:
Maybe unnecessary, but I think if folks hitting link errors it's better to swarm than struggle.
```suggestion
Verify that all automated checks pass before proceeding. If you encounter any linking failures and need help resolving them, reach out in the typical docs channels.
```